### ISSUE TITLE
Document ability to change user-agent

### ISSUE THREAD

**Issue by FezVrasta** [matthew-andrews/isomorphic-fetch#116]
When you run `isomorphic-fetch` from node, you can define a custom `User-Agent` string.
When you run it from a browser, you are not allowed to define a custom one.

I think this should be documented somewhere because it can lead to confusion.

**Comment by shenders13** [comment]
I find it hard to believe that set a User-Agent header from the Browser using fetch. Is that really the case? Does AJAX (or another library) allow you to set the User-Agent header from the browser?

**Comment by FezVrasta** [comment]
Nope, but since on node it works, it would be nice to document this difference between the two environments
